### DISCUSSION TITLE
Add mappings to re-order commits in the rebase todo buffer?

### DISCUSSION THREAD

**Discussion by davidatsurge** [NeogitOrg/neogit#1029] [Q&A] [answered]
After I updated my neogit recently, mappings were added to the rebase todo buffer that override very commonly (at least by me) used keys in normal mode, like `p` and `d`. This made re-ordering commits very unnatural for me since I had to resort to excercising vim-fu and finding unbound keys to accomplish what I did with `dd` and `p`.

Can I suggest adding mappings to do reordering? Or maybe there already is an easy way to re-order commits in neogit I haven't found out about ...

**Comment by CKolkey** [comment, marked as answer]
Ah, of course. I've had `<m-j>` and `<m-k>` bound to move a line up and down for so long I forgot that wasn't stock. 

I'll add those, but also just make the bindings configurable for users.

**Reply by davidatsurge** [reply]
Thanks for the quick reply! That sounds good to me. Btw, is there a way to customize (or entirely disable) these new mappings (ie, including the ones to fixup/squash/pick/etc).

**Reply by CKolkey** [reply]
Yea! Forgot to update the README, but you can remap the commands the same as everything else:

```lua
  setup({
    mappings = {
      commit_editor = {
        ["q"] = "Close",
        ["<c-c><c-c>"] = "Submit",
        ["<c-c><c-k>"] = "Abort",
      },
      rebase_editor = {
        ["p"] = "Pick",
        ["r"] = "Reword",
        ["e"] = "Edit",
        ["s"] = "Squash",
        ["f"] = "Fixup",
        ["x"] = "Execute",
        ["d"] = "Drop",
        ["b"] = "Break",
        ["q"] = "Close",
        ["<cr>"] = "OpenCommit",
        ["gk"] = "MoveUp",
        ["gj"] = "MoveDown",
        ["<c-c><c-c>"] = "Submit",
        ["<c-c><c-k>"] = "Abort",
      },
    }
  })
```

To disable a mapping, you can, for example, set the value of something to `false`:
`["<c-c><c-k>"] = false,`
